GETOOLS_SOURCE/modules/Overlappy.py

```diff
@@ -125,11 +125,16 @@ class OverlappySettings:
 	rangeOffsetZ = (float("-inf"), float("inf"), 0, 100)
 	
 class Overlappy:
-	version = "v2.8"
+	version = "v2.9"
 	name = "OVERLAPPY"
 	title = name + " " + version
 
-	def __init__(self):
+	# HACK use only for code editor # TODO try to find better way to get access to other classes with cross import
```

Comment:
This should be changed as soon as possible. Now it works fine, but it could be unpredictable in future